kvdb/etcd/bucket.go

```diff
@@ -130,3 +130,10 @@ func ValueKey(key string, buckets ...string) string {
 
 	return string(makeValueKey(bucket, []byte(key)))
 }
+
+// SequenceKey is a helper function used in tests to create a sequence key from
```

Comment:
Trying to understand the use case here, so it's used in tests by other repo?

Reply:
Yeah, the comment here isn't fully accurate. We need this function in the [migration tool](https://github.com/lightninglabs/lndinit/pull/21) in order to re-create the correct destination keys in `etcd` for the sequences.